README.md

```diff
@@ -84,6 +84,20 @@ end
 song.to_json #=> {"name":"Fallout","track":1}
 ```
 
+You can also a naming strategy at a Representer-level:
+
+```ruby
+module SongRepresenter
+  include Representable::JSON
+
+  self.naming_strategy = ->(name) { name.camelize } # (with ActiveSupport available)
+
+  property :song_title
+  property :song_track, as: :track
+end
+
+song.to_json #=> {"songTitle":"Fallout","track":1}
```

Comment:
If you were using the `camelize` of `ActiveSupport` (as indicated), it would read `SongTitle` and `Track`.

Either use `camelize(:lower)` as example code above or change the JSON here to upper camel case.